### DISCUSSION TITLE
guide/write/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#138] [docs-comment]
# guide/write/

提示 以下内容中，以 sourceDir 表示 项目中用于保存 Markdown 文件的目录。 如 package.json的scripts 中配置的 vuepress dev src，则表示 sourceDir 为 src 目录 约定 使用本主题编写文章是一件很轻松的事情，你可以在 sourceDir目录中按照你的个人命名喜好新建任意名字的Markd...

https://plume.pengzhanbo.cn/guide/write/

**Comment by tianhairui** [comment]
博主好，在`约定`段落中最后一行，超链接 `主题配置notes` 报404，地址可能有误，请确认下。

**Reply by pengzhanbo** [reply]
感谢指出，已修改

**Comment by callMeExiao** [comment]
永久链接-notes-“作为 ntoe 文章的永久链接”
ntoe拼错了吧？
